docs/future-architecture/architecture/09-templates.md

```
@@ -0,0 +1,112 @@
+  security:
+    runAsNonRoot: false           # sysbox/kata allow safe root
+    seccompProfile: RuntimeDefault
+    dropCapabilities: ["ALL"]
+    addCapabilities: []           # template can request specific caps with justification
+```
+
+## Tenant → template mapping
+
+L4 resolves at session creation:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)